packages/server/src/organization/organization.entity.ts

```diff
@@ -36,15 +37,15 @@ export class OrganizationModel extends BaseEntity {
 
   @Exclude()
   @OneToMany(
-    type => OrganizationUserModel,
-    organizationUser => organizationUser.organization,
+    (type) => OrganizationUserModel,
+    (organizationUser) => organizationUser.organization,
   )
   organizationUsers: OrganizationUserModel[];
 
   @Exclude()
   @OneToMany(
-    type => OrganizationUserModel,
-    organizationUser => organizationUser.organization,
+    (type) => OrganizationUserModel,
+    (organizationUser) => organizationUser.organization,
```

Comment:
Change to `OrganizationCourseModel`
add join to set up FK

Reply:
Good catch